tv_apps/app-deck/js/app_deck.js

```diff
@@ -276,6 +334,15 @@
       if (focused && focused.dataset && focused.dataset.manifestURL) {
         Applications.launch(
           focused.dataset.manifestURL, focused.dataset.entryPoint);
+      } else if (focused && focused.dataset && focused.dataset.url) {
+        /* jshint nonew: false */
```

Comment:
According to the final decision, let's use `window.open(url, '_blank', 'remote=true');` to open this kind of bookmarks.